tests/integration_tests/adapters/interactive_brokers/conftest.py

```diff
@@ -73,9 +73,9 @@ def exec_client_config():
 
 
 @pytest.fixture()
-def ib_client(data_client_config, loop, msgbus, cache, clock, logger):
+def ib_client(data_client_config, event_loop, msgbus, cache, clock, logger):
```

Comment:
Small nit but the event loop param is very typically `loop`.

Reply:
Will revert!